aflr/api_resources/file.py

```diff
@@ -1,44 +1,19 @@
-from posixpath import basename
-import aflr
-from aflr.api_request import APIRequest
-import os
+from aflr.helper_classes import UploadableResource, ListableResource
 
 
-class File(APIRequest):
+class File(UploadableResource, ListableResource):
     custom_audio_resource_path = "/file/customaudio/"
```

Comment:
please rename to `upload_path`

Reply:
done

Reply:
dont like doing this, since the `custom_audio_resource_path` is also getting used by the 2 File Methods `get_download_url` and `list`

Reply:
name can be changed. actually there is one more change to be made here:
why have this? `return cls._get_request( path_param=cls.upload_path + "tags?tags=" + tags )`
We support path parameters on the `_get_request()` method. try doing something like
`return cls._get_request( path_param=cls.upload_path, request_params=tags )`

Reply:
i see